src/components/helpers/formulas.ts

```diff
@@ -0,0 +1,23 @@
+import { UID } from "../..";
+import { rangeTokenize } from "../../formulas";
+import { Getters } from "../../types/getters";
+
+export function adaptFormulaToSheet(
```

Comment:
nitpick: maybe could be a getter in range ?

Reply:
There are others functions that affect formulas coming to this file (in others PRs). I feel like range is already big enough and doesn't need more bloating.